### ISSUE TITLE
Job 26:7 UDB north/earth

### ISSUE THREAD

**Issue by SusanQuigley** [Door43/udb-en#194]
UDB: God stretches out the **earth** over the huge empty space

Could "earth" be changed to "heavens" or "skies"?

The TFT and UDB are the only versions I see that translate "north" as "earth."
NICOT explains convincingly that the north refers to the heavens.
God's Word to Nations has "his heavens."
Some versions keep the word "north" but add the idea of skies: "northern skies"
NET Bible note: " it may just represent the northern skies – the stars"

**Comment by TomWarren** [comment]
The UDB now reads:
\v 7 God stretches out his reach over the stars of the northern sky 
\q1 and over the huge empty space,
\q2 and there he places the earth and its rest upon nothing.

The ULB now reads: 
\v 7 He stretches out over the northern sky 
\q and over the empty space,
\q and he hangs the earth upon nothing.

Poetry! is tough.

**Comment by SusanQuigley** [comment]
The new UDB does not yet show up in Github.
ULB: and he hangs the earth upon nothing.
This is ambiguous to me. Does it mean that there is nothing above holding the hanging earth, or that the earth is hanging over nothing?

The new ULB and UDB sound like God stretches out himself or his arm. Is that what you want?
What would you think of these?

ULB: He stretches out the north over the empty space
and he hangs the earth  over nothing.

UDB: God stretches out the northern sky over  the huge empty space,
and there he places the earth and it rests upon nothing.

**Comment by TomWarren** [comment]
The HEB is a set of terms linked: עַל־בְּלִי־מָֽה׃ lit  "on-not-whatever." 
The NET has "he suspends the earth on nothing."
This reminds me of Old Testament at Belhaven with John Akers (1969) and he spoke of how before God made nothing, nothing didn't exist. 

The Heb is "he hangs (perhaps suspends) the earth on-not-whatever".
I think that the point that you are wrestling with is what is in view, that he hung the world on nothing. And so, I think the ULB should stand.

I reworked the UDB:
The UDB of JOB 26;7 now reads: 

\v 7 God stretches out his reach over the northern sky 
\q1 and over the huge empty space,
\q2 and he places the earth in that huge empty space, but it is not resting on anything.

Is that ok? 

Thanks,
Tom

**Comment by SusanQuigley** [comment]
I guess so, but I'll need help to understand it so I can write notes.

What does it mean for God to stretch out over the northern sky?
 Or for him to stretch out his reach over the northern sky?
Is he lying down on the sky? Is he reaching over the sky for something just beyond it?
Are these images used to imply that he is greater than the sky?

**Comment by TomWarren** [comment]
If seems that God is reminding Job of his absolute greatness, that he can reach out to the farthest extent of the sky and touch it.  I think this is a way of his displaying his "immensity" compared with nature. God is infinite, and the greatest expanse of the sky, as huge as it is, is still finite. 
I am not sure that you would be able to explain all that God is telling us about himself. We are, after all, finite like the rest of the universe. And God is infinite. 
It would not be that God should wants us to compare him to the sky, but that he is above and beyond even all the stars and the farthest reaches of the universe. 
Big stuff, this.

**Comment by SusanQuigley** [comment]
Ok. I'll write a note based on that. Thank you.

**Comment by SusanQuigley** [comment]
I wrote a note about it in line with your explanation. 
But if people have access to any other Bible, they may wonder why the ULB and UDB are so different from their Bibles. So here are the notes I wrote.

**He stretches out over the northern sky and over the empty space** - The sky and empty space are great, but God is greater. AT: "God is greater than the northern sky and empty space over which he stretches" (See: Metaphor)
**He stretches out over the northern sky and over the empty space** - Some versions have "He stretches out the north over empty space" or "He spreads out the heavens over empty space."
